tests/tests/test_mender_connect.py

```diff
@@ -398,14 +415,24 @@ def docker_env(self, enterprise_no_client_class):
 
         yield env
 
+    @pytest.fixture(scope="function")
+    def docker_env_function(self, enterprise_no_client):
+        env = enterprise_no_client
+
+        device, devconn = connected_device(env)
+
+        env.device = device
+        env.devconnect = devconn
+
+        yield env
+
 
 class TestRemoteTerminalEnterprise_1_0(_TestRemoteTerminalBase):
     """
     This set of tests uses mender-connect v1.0
     """
 
-    @pytest.fixture(autouse=True, scope="class")
-    def docker_env(self, request):
+    def docker_env_impl(self, request):
```

Comment:
> nit: _class suffix? (see comment below)

Well, here the `_class_  suffix does not apply because this method is not a fixture at all.

> also what stands behind the impl suffix?

Guess no more! impl stands for implementation. I will leave it as is because it is a common short word we use across the test framework, see [this](https://github.com/mendersoftware/integration/blob/3e6386fbb42042dd72b3738e5ec60aebe4f4c85e/tests/common_setup.py#L58-L81) or [this](https://github.com/mendersoftware/integration/blob/3e6386fbb42042dd72b3738e5ec60aebe4f4c85e/testutils/infra/device.py#L316-L342) for some examples.